app/models/supplejack_api/support/harvestable.rb

```diff
@@ -85,7 +85,7 @@ def flush_old_records(source_id, job_id)
             :'fragments.source_id' => source_id,
             :'fragments.job_id'.ne => job_id,
             :status.in => %w[active supressed]
-          ).update_all(status: 'deleted', updated_at: Time.now)
+          ).update_all(status: 'deleted', updated_at: Time.now, '$set': { 'fragments.$.job_id' => job_id })
```

Comment:
Is this happening for all of the fragments or just the primary fragment?

Reply:
all fragments